### PR TITLE
Update Security Group EGress

### PR DESCRIPTION
**Pull request by jdrowe**

### CODE REVIEW

File: provisioning/main.json [hunk 1]
```diff
@@ -189,6 +189,56 @@
...
+            "IpProtocol" : "tcp",
+            "FromPort" : "2222",
+            "ToPort" : "2222",
+            "CidrIp" : "52.22.14.222/32"
+          },
+          {
+            "IpProtocol" : "udp",
+            "FromPort" : "53",
+            "ToPort" : "53",
+            "CidrIp" : "10.9.0.0/16"
```
Comment (dlapiduz) [inline provisioning/main.json]: @jdrowe I don't think we need 10.9.x.x in prod, just 10.10...

**Comment by jdrowe** [comment]
@dlapiduz Updated

**Comment by dlapiduz** [comment]
@jdrowe lets do these changes:
- udp:53 0.0.0.0/0 allowed
- tcp:53 0.0.0.0/0 allowed
- all:all 10.10.0.0/16 allowed

**Comment by LinuxBozo** [comment]
no longer relevant.. managed with terraform..

**Pull request closed without merging**